db/hints/internal/hint_endpoint_manager.cc

```diff
@@ -43,6 +42,8 @@ namespace {
 
 constexpr std::chrono::seconds HINT_FILE_WRITE_TIMEOUT = std::chrono::seconds(2);
 
+logging::logger hint_logger{"hint_endpoint_manager"};
```

Comment:
It's going to be quite frustrating to tweak all those loggers' levels with the CLI, if needed. In the next patch you add "prefixes" to log messages anyway, why not extend that practice and add relevant "prefixes" here to tell different files' messages from each other?

It's not merge blocker comment though